catkin_ws/src/kpr_testing/src/valves.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+
+import rospy, termios, sys, contextlib, os
+import numpy as np
+from ur_msgs.srv import *
+from ur_msgs.msg import *
+
+header = """
+Check the status of I/O-pins
+\t-^D to exit
+\t-i to change income settings
+\t-p to send io states
+\t-t to toggle service succes
+"""
+
+def display():
```

Comment:
Please add comments about the functioning, parameters and return values of each function